src/components/Filter.res

```diff
@@ -288,12 +288,82 @@ let make = (
     setfilterKeys(_ => keys)
   }
 
+  let allFiltersUI =
+    <Menu \"as"="div" className="relative inline-block text-left">
+      {_ =>
+        <div>
+          <Menu.Button
+            className="flex items-center whitespace-pre leading-5 justify-center text-sm  px-4 py-2 font-medium rounded-lg h-10 hover:bg-opacity-80 bg-white border">
+            {_ => {
+              <>
+                <Icon className={"mr-2"} name="plus" size=15 />
+                {"Add Filters"->React.string}
+              </>
+            }}
+          </Menu.Button>
+          <Transition
+            \"as"="span"
+            enter="transition ease-out duration-100"
+            enterFrom="transform opacity-0 scale-95"
+            enterTo="transform opacity-100 scale-100"
+            leave="transition ease-in duration-75"
+            leaveFrom="transform opacity-100 scale-100"
+            leaveTo="transform opacity-0 scale-95">
+            <Menu.Items
+              className="absolute left-0 w-fit z-50 mt-2 origin-top-right bg-white dark:bg-jp-gray-950 divide-y divide-gray-100 rounded-md shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
+              {_ => {
+                <>
+                  <div className="px-1 py-1">
+                    {allFilters
+                    ->Array.mapWithIndex((option, i) =>
+                      <Menu.Item key={i->Int.toString}>
+                        {props =>
+                          <div className="relative w-max">
+                            <button
+                              onClick={_ => addFilter(option)}
+                              className={
+                                let activeClasses = if props["active"] {
+                                  "group flex rounded-md items-center w-48 px-2 py-2 text-sm bg-gray-100 dark:bg-black"
+                                } else {
+                                  "group flex rounded-md items-center w-48 px-2 py-2 text-sm"
+                                }
+                                `${activeClasses} font-medium`
+                              }>
+                              <RenderIf condition={option.label->LogicUtils.isNonEmptyString}>
+                                <div className="mr-5">
+                                  {option.label->LogicUtils.snakeToTitle->React.string}
+                                </div>
+                              </RenderIf>
+                              <RenderIf condition={option.label->LogicUtils.isEmptyString}>
+                                <div className="mr-5">
+                                  {option.inputNames
+                                  ->LogicUtils.getValueFromArray(0, "")
+                                  ->LogicUtils.snakeToTitle
```

Comment:
This computation can be moved outside

Reply:
its a very small computation, i dont think we really need to separate it out